nncf/experimental/tensorflow/graph/node_attributes.py

```diff
@@ -0,0 +1,33 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import List
+
+from nncf.common.graph.layer_attributes import BaseLayerAttributes
+
+
+class TFNodeAttributes(BaseLayerAttributes):
+    def __init__(self, data_format: str):
+        self._data_format = data_format
+
+    def get_data_format(self) -> str:
+        return self._data_format
+
+
+class TFWeightedNodeAttributes(TFNodeAttributes):
```

Comment:
Documentation missing

Reply:
Thanks for the comment! I added missing docstrings.